### ISSUE TITLE
A11y P2 - First item in Recent Episodes component not always read out

### ISSUE THREAD

**Issue by LilyL0u** [bbc/simorgh#8354]
This was discovered on NVDA on Windows 7 a11y laptop

When using TAB to navigate links, the first link in the recent episodes is not always read out (silence). When I first go to the page the first page IS reading the first link, unhighlighted, along with ‘landmark list with three items’. However nothing is announced on this element when it is focused on when you go back to the page frame and then to the first item again. 

I have a video of the behaviour but to make the file small enough I need to reduce the size and it is crashing Preview...

This behaviour can also be seen in 2bitstudio example 4 with links http://2bitstudio.co.uk/code-tests/article/list-ordered/ordered-list.html

**Comment by LilyL0u** [comment]
@greenc05 As this behaviour is the same on the recent episodes component and your 2bitstudio page, does this mean it is something we don't need to fix/ is a low priority to fix?

The slack thread about this bug is here https://bbcnews.slack.com/archives/CP14D941L/p1605191147000200 
It has videos of the behaviour

**Comment by gavinspence** [comment]
Retested with NVDA with the component on the page and the first item is read out every time. Reads "list with 4 items, [content of the link]"

**Comment by greenc05** [comment]
When testing this myself on Windows 7 and navigating via the 'tab' key between the links in the storybook component, the first time I did this it just read out 'Audio' for the first link, the rest of the links were as expected. When reloading the page it was fine. Though I have just re-tested and I experienced the 'empty' tab as per Lily. Let's re-test this when we've added the role text span to fix another NVDA bug.

**Comment by LilyL0u** [comment]
I am still experiencing part of this bug in storybook, Firefox, NVDA. @jroebu14 

it is now always announcing the first link when I first visit it, but it still is silent when I navigate back to the page frame, and then to the first episode item again.

This is also happening on the 2bitstudio link.

**Comment by gavinspence** [comment]
Can we recheck this on the page? https://www.test.bbc.com/indonesia/bbc_indonesian_radio/programmes/w13xtt0s

I've tested this with NVDA on the a11y laptop and it seems to work as expected.

**Comment by LilyL0u** [comment]
I realise that I don't think this will affect the behaviour of the recent episodes component on the page as the top episode item is not the first focusable item on the page. I have only seen it happen on the first focusable item on a page, which it was within storybook, but on the page there are other focusable items first. So I think we can close this.
